<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QMainWindow" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="lwt_File">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="let_Menu">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>16</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Copy">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Exit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Paste">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Enter">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Delete">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_UDisk">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>UDisk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cut">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Root">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Root</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Up">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
